env0/resource_environment.go

```diff
@@ -1126,8 +1132,7 @@ func getDeployPayload(d *schema.ResourceData, apiClient client.ApiClientInterfac
 	var err error
 
 	if isTemplateless(d) {
-		templateId, ok := d.GetOk("without_template_settings.0.id")
-		if ok {
+		if templateId, ok := d.GetOk("without_template_settings.0.id"); ok {
```

Comment:
no change... just one less line.

Reply:
this syntax is super strange ... do we use it a lot in this codebase / is it common in Go?

Reply:
Yes.
This is common go.